tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateCoverageLevels.java

```diff
@@ -105,7 +105,7 @@ private static void showEnglish(PrintWriter out) throws IOException {
         String locale = "en";
         Set<String> sorted =
                 Builder.with(new TreeSet<String>())
-                        .addAll(cldrFile.iterator())
+                        .addAll(cldrFile.iteratorDefault())
                         .addAll(cldrFile.getExtraPaths())
```

Comment:
note: if iteratorDefault includes extra paths, then getExtraPaths would be superfluous here, except that iteratorDefault skips paths with null values (getStringValue(path) == null)

alternatively fullIterator() could be called here